src/sarif/cmdline/main.py

```diff
@@ -86,13 +86,22 @@ def _create_arg_parser():
         + "(or for diff, an increase in issues at that level).",
     )
 
-    for cmd in ["blame", "csv", "summary", "html", "word"]:
+    for cmd in ["blame", "csv", "html", "summary", "word"]:
         subparser[cmd].add_argument(
             "--output", "-o", type=str, help="Output file or directory"
         )
     for cmd in ["diff", "ls", "trend", "usage"]:
         subparser[cmd].add_argument("--output", "-o", type=str, help="Output file")
 
+    for cmd in ["csv", "diff", "summary", "html", "trend", "word"]:
```

Comment:
You put the other lists into alphabetical order but not this one
```suggestion
    for cmd in ["csv", "diff", "html", "summary", "trend", "word"]:
```

Reply:
Good spot - fixed :-)